validation/linux_readonly_paths.go

```diff
@@ -21,6 +21,12 @@ func main() {
 		if err != nil {
 			return err
 		}
+		// create a temp file to make testDir non-empty
+		tmpfile, err := ioutil.TempFile(testDir, "tmp")
+		if err != nil {
+			return err
+		}
+		defer tmpfile.Close()
```

Comment:
I didn't understand why do that. I tested it as if it had no effect on the results.

Reply:
@q384566678 You're right.
What I wanted to do was `defer os.Remove(tmpfile.Name())`. Fixed it.